requests_async/adapters.py

```diff
@@ -92,3 +103,7 @@ class HTTPAdapter(requests.adapters.HTTPAdapter):
         )
 
         return self.build_response(request, resp)
+
+    async def _get_connection(self, hostname, port, **conn_kwargs):
```

Comment:
It doesn't look to me like you need this separate function - just use `asyncio.open_connection` directly instead.

Reply:
Okay I will fix that)